backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/migrations/V0028_Add_Application_Status.kt

```diff
@@ -0,0 +1,23 @@
+package app.ehrenamtskarte.backend.migration.migrations
+
+import app.ehrenamtskarte.backend.migration.Migration
+import app.ehrenamtskarte.backend.migration.Statement
+
+@Suppress("ClassName")
+internal class V0028_Add_Application_Status : Migration() {
+    override val migrate: Statement = {
+        exec(
+            """
+            -- Add column "status"
+            alter table "applications" add column "status" varchar(32) not null default 'Pending'
+                 check ("status" in ('Pending', 'Approved', 'ApprovedCardCreated', 'Rejected'));
+            -- Set "status" to 'AcceptedCardCreated' for rows that have "cardCreated" set to true
+            update "applications" set "status" = 'AcceptedCardCreated' where "cardCreated";
+            -- Remove ”cardCreated" column
+            alter table "applications" drop column "cardCreated";
+            -- Add column "statusResolvedDate"
+            alter table "applications" add column "statusResolvedDate" timestamptz default null;
```

Comment:
❓ why is `timestamptz` needed here. I think we only store UTC in the database

Reply:
hm okay i understand. as long as we don't have issues with mixing the types when we display it for the users, i'm fine with that. Even it looks like we can not convert the current `timestamp` to `timestampz`

Reply:
In fact, exposed [only added support for this two years ago](https://github.com/JetBrains/Exposed/pull/1787). That's why most of our timestamps do not use this type.
While the name ("timestamp with timezone") is more than confusing to me, it would be the better to have all our timestamps use that column type.
I think the old timestamps are the reason why we have this weird workaround where we set the system time zone to UTC on startup.